io/src/hdl_grabber.cpp

```diff
@@ -565,7 +564,7 @@ pcl::HDLGrabber::stop ()
 bool
 pcl::HDLGrabber::isRunning () const
 {
-  return (!hdl_data_.isEmpty () || (hdl_read_packet_thread_ != NULL && !hdl_read_packet_thread_->timed_join (boost::posix_time::milliseconds (10))));
+  return (!hdl_data_.isEmpty () || (hdl_read_packet_thread_ != NULL));
```

Comment:
the timed_join function doesn't exist in Standard C++ (and is also deprecated in boost). I didn't really understand that purpose of that check. Maybe someone can clarify this?